### PR TITLE
Add Slicer

### PR DESCRIPTION
**Pull request by maxdanwitz**

### CODE REVIEW

File: pvutils/scripts/slicer.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+"""
+Slices a 3d geometry with a series of planes.
+"""
```
[the remark this answers is not in the document]
Comment (maxdanwitz) [reply pvutils/scripts/slicer.py]: Thanks for the review, both points were added.

**Comment by maxdanwitz** [comment]
> @maxdanwitz thanks for contributing your script to this repository. One thing I am not sure of is where to put the script. It is not really suited for the `scripts` folder, since the scripts there are supposed to be called from within python. Maybe we add a `free_floating_scripts` folder to the root directory? Any opinion?

@isteinbrecher thanks for the review. I moved the script to the suggested folder.

File: pvutils/scripts/slicer.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+# Python imports.
+import numpy as np
+from pathlib import Path
+
+# ParaView imports.
+import paraview.simple as pa
+
+# Define slicing
+start_point = 0.0
+end_point = 1.0
```
[the remark this answers is not in the document]
Comment (maxdanwitz) [reply pvutils/scripts/slicer.py]: To employ this script in a meaningful way, the user needs to know the geometry. Therefore, I think the additional freedom (slicing only parts of the geometry) outweighs the complication of specifying start and end point.

**Comment by maxdanwitz** [comment]
Screenshot 2023-05-08 at 16 32 24

It seems your black is not my black. So hard to argue about colours...

**Comment by isteinbrecher** [comment]
With a recent release of `black`, the change you noticed was added. I think the current version is the correct one.

**Comment by isteinbrecher** [comment]
Changes look good to me.

**Review by isteinbrecher — APPROVED** [review]

**Pull request merged**